public/app/plugins/panel/geomap/utils/location.ts

```diff
@@ -147,12 +147,12 @@ export interface LocationInfo {
   points: Point[];
 }
 
-export function dataFrameToPoints(frame: DataFrame, location: LocationFieldMatchers): LocationInfo {
+export async function dataFrameToPoints(frame: DataFrame, location: LocationFieldMatchers): Promise<LocationInfo> {
   const info: LocationInfo = {
     points: [],
   };
   if (!frame?.length) {
-    return info;
+    return Promise.resolve(info);
```

Comment:
should just be able to `return info;` here as before? same on line 179.

Reply:
nice! I must have learned something long ago that is not quite right!